src/components/collapsible-section/collapsible-section.scss

```diff
@@ -13,17 +20,15 @@
 }
 
 section {
-    background-color: #ffffff;
```

Comment:
This was stupid. Clearfix is the right way to do it. Bad past-Adrian!

Reply:
To clarify, I had to remove this, because with semitransparent background for the header and body, this effectively "reset" the "base-background", resulting in nested collapsible-sections not appearing as expected, but like this instead:

![image](https://user-images.githubusercontent.com/154725/73429624-68964a00-433c-11ea-96a7-f0318d034863.png)